keymaster/ta/mbedtls_proxy.c

```diff
@@ -272,8 +270,39 @@ static keymaster_error_t convert_epoch_to_date_str(uint32_t sec,
 	snprintf((char *)(t_str + 8), 3, "%02u", ntp_hour);
 	snprintf((char *)(t_str + 10), 3, "%02u", ntp_minute);
 	snprintf((char *)(t_str + 12), 3, "%02u", ntp_second);
+
 	DMSG("seconds since epoch: %" PRIu32, sec);
 	DMSG("Date string: %s", t_str);
+	switch (ntp_week_day) {
+	case 0:
+		DMSG("Sunday");
+		break;
+	case 1:
+		DMSG("Monday");
+		break;
+	case 2:
+		DMSG("Tuesday");
+		break;
+	case 3:
+		DMSG("Wednesday");
+		break;
+	case 4:
+		DMSG("Thursday");
+		break;
+	case 5:
+		DMSG("Friday");
+		break;
+	case 6:
+		DMSG("Saturday");
+		break;
+	default:
+		break;
+	}
+	if (leap_year_ind) {
+		DMSG("%04u is a leap year", ntp_year);
+	} else {
+		DMSG("%04u is not a leap year", ntp_year);
+	}
```

Comment:
Any specific reason for adding so many debug messages in this patch ?

Reply:
This is wrt https://github.com/linaro-swg/kmgk/pull/21 61661f5ba870a346ff32fb3231cc8621f2c985c6. The warning 'variable set but not used' reappears when log level is < DBG, so use the variable itself to determine which debug logs to print.